src/modules/commentFunctions.js

```diff
@@ -0,0 +1,56 @@
+import commentsCounter from './commentsCounter.js';
+import { baseUrl, involvementAppId } from './involvementApi.js';
+
+const commentUrl = `${baseUrl}apps/${involvementAppId}/comments`;
+const commentsList = document.querySelector('.comments-list');
+const getComment = async (id) => {
+  try {
+    const response = await fetch(`${commentUrl}?item_id=${id}`);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw new Error(error);
+  }
+};
+
+const createComment = (item) => {
+  const comment = document.createElement('li');
+  comment.textContent = `${item.creation_date} ${item.username}: ${item.comment}`;
+  commentsList.appendChild(comment);
+};
+const postComment = () => {
+  const nameInput = document.querySelector('.name-input');
+  const commentInput = document.querySelector('.comment-input');
+  const submitCommentBtn = document.querySelector('.submit-comment-btn');
+  const submitCommentHandler = async (e) => {
+    e.preventDefault();
+    const id = Number(e.target.id);
+    const username = nameInput.value;
+    const comment = commentInput.value;
+    if (username && comment) {
+      const data = { item_id: id, username, comment };
+      fetch(commentUrl, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      }).then(() => {
+        nameInput.value = '';
+        commentInput.value = '';
+        getComment(id).then((arr) => {
+          commentsList.innerHTML = '';
+          if (arr.length > 0) {
+            arr.forEach((item) => {
+              createComment(item);
+              commentsCounter();
+            });
+          }
+        });
+      });
+    }
+  };
+  submitCommentBtn.addEventListener('click', submitCommentHandler);
```

Comment:
You guys did a great job! Perfectly implemented code!

- [ ] All loading movies, likes are well-implemented. However, I got one issue in your comments section. Let me explain, what I'm getting. First I opend Batman movie then left a comment in it. Then I tried to open other movie but somehow the comment that I left in Batman is visible in other movie as well. Also, the comments are keeps adding from other movies comments. Please, fix the issue. Thank you!

Here is the video about the issue:

https://user-images.githubusercontent.com/69011963/233781228-d7838704-94db-4416-89ee-e97330f2d268.mov

Reply:
Thanks for the review
Issue fixed